icrawler/downloader.py

```diff
@@ -5,6 +5,7 @@
 import threading
 
 import requests
+from urllib.parse import urlparse
```

Comment:
In fact I use python 3 too and hope that python 3 becomes more popular so I do not need to consider the compatibility with python 2, but there seems to be still a long way to go. Anyway, thanks a lot for your help!